DESCRIPTION

```diff
@@ -27,7 +27,9 @@ Imports:
     ggplot2,
     magrittr,
     rlang,
-    scales
+    scales,
+    SGL,
+    gglasso
```

Comment:
The package shouldn't depend on `{SGL}` or `{gglasso}`

Reply:
These two packages remain on the DESCRIPTION for verifying test results only. I'll remove them when I remove all the tests related to them.